docs/setup/prerequisites/kubernetes.md

```diff
@@ -8,3 +8,20 @@ KubeEdge is designed to extend Kubernetes from cloud to edge, and while it's bui
 Kubernetes acts as a foundation for KubeEdge, which extends Kubernetes' capabilities to edge computing. KubeEdge leverages the core functionalities of Kubernetes, enabling it to manage workloads not just in a centralized cloud environment, but also across various edge locations. This extension includes adapting Kubernetes to the unique requirements of edge computing, such as network unreliability and resource constraints.
 
 In summary, while a deep understanding of Kubernetes is highly beneficial for working with KubeEdge, it's also necessary to have a Kubernetes control plane in place. This control plane is crucial for KubeEdge to function, as it extends the native Kubernetes capabilities to edge devices, allowing them to run, orchestrate, and manage workloads in edge environments.
+
+## Kubernetes compatibility
```

Comment:
Displaying Kubernetes compatibility in multiple places will increase maintenance costs. I think it's more appropriate to add the link https://github.com/kubeedge/kubeedge?tab=readme-ov-file#kubernetes-compatibility here or add a link at readme of kubeedge/kubeedge